scripts/docs-generation/wdioDocs.js

```diff
@@ -35,12 +43,11 @@ exports.generateWdioDocs = async (sidebars) => {
 
             const filepath = path.join(COMMAND_DIR, scope, file)
             const outputpath = path.join(docDir, `_${file.replace(/(js|ts)/, 'md')}`)
-            const out = await processDocs({data:[filepath],template:TEMPLATE_PATH })
+            const out = await processDocs({data:[filepath],template:TEMPLATE_PATH, options:JSDOC_OPTIONS}) 
```

Comment:
Are you sure both packages have the same interface?

Reply:
Yeah! The build is successfully compiled and the ```.md``` files are generated without any errors and warnings

Reply:
it is failing for me, see comment below